### PR TITLE
10068 story 

### PR DESCRIPTION
**Pull request by ujahio**
Story: [10068](https://github.com/flexion/ef-cms/issues/10068)

### CODE REVIEW

**Review by jasonfu9966 — APPROVED** [review]

**Comment by TomElliottFlexion** [comment]
May I suggest either pulling test into this branch, or closing and re-opening the PR? That way we wouldn't see the extra changes

File: web-client/src/presenter/computeds/formattedDocketEntries.ts
Comment (jasonfu9966) [inline web-client/src/presenter/computeds/formattedDocketEntries.ts:349]: We're adding this compute here, but we noticed that there is similar work going into the `DocketEntry` entity through a PR that is currently awaiting approvals to go to `test`

We would prefer it if we could use the new work going to the `DocketEntry` entity but we can't realistically get that work until it's merged into staging.

Open to thoughts or suggestions from reviewers on how to handle this situation.

File: web-client/src/presenter/sequences/batchDownloadDocketEntriesSequence.ts
Comment (TomElliottFlexion) [inline web-client/src/presenter/sequences/batchDownloadDocketEntriesSequence.ts:10]: Nit: I'm not sure we need an entire cerebral conditional to check this variable: we could combine `addPrintableDocketRecordCheckAction` and `generateDocketRecordPdfUrlAction`

File: web-client/src/views/DocketRecord/DocketRecord.tsx
Comment (TomElliottFlexion) [inline web-client/src/views/DocketRecord/DocketRecord.tsx]: Not super important: since we don't have a React testing framework, ideally most of the display logic would be done in a computed. I don't know if it's possible to bind documentsSelectorHeaderInput.indeterminate to state, but that would allow this logic to be tested

**Review by TomElliottFlexion — APPROVED** [review]

File: shared/src/proxies/documents/batchDownloadDocketEntriesProxy.ts
Comment (nechama-krigsman) [inline shared/src/proxies/documents/batchDownloadDocketEntriesProxy.ts:10]: Should this endpoint specify it's downloading docket entries?

File: shared/src/business/useCases/document/batchDownloadDocketEntriesInteractor.ts
Comment (nechama-krigsman) [inline shared/src/business/useCases/document/batchDownloadDocketEntriesInteractor.ts:126]: is this number always set to zero?

File: cypress/cypress-integration/integration/batch-download-documents.cy.ts
Comment (nechama-krigsman) [inline cypress/cypress-integration/integration/batch-download-documents.cy.ts:1]: There are a couple of todo comments and some code that is commented out in this file. were those addressed or can they be removed?

**Review by nechama-krigsman — APPROVED** [review]

File: shared/src/business/useCases/document/batchDownloadDocketEntriesInteractor.ts
Comment (ujahio) [reply shared/src/business/useCases/document/batchDownloadDocketEntriesInteractor.ts:126]: The batch download operation is written to download cases and documents associated to those cases. For this story, we are only downloading documents on a case, hence, 0 for case downloads.

File: shared/src/proxies/documents/batchDownloadDocketEntriesProxy.ts
Comment (ujahio) [reply shared/src/proxies/documents/batchDownloadDocketEntriesProxy.ts:10]: We are specifically downloading the documents "associated" with the docket entries. We aren't downloading the docket entries's data itself.

**Pull request merged**
